sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/constants/AadJwtClaimNames.java

```diff
@@ -10,7 +10,10 @@
  * @see <a href="https://docs.microsoft.com/azure/active-directory/develop/access-tokens">Access tokens</a>
  * @see <a href="https://docs.microsoft.com/azure/active-directory/develop/id-tokens">ID tokens</a>
  */
-public class AadJwtClaimNames {
+public final class AadJwtClaimNames {
```

Comment:
Adding private ctor trigger check rule that "final" should be declared. Same for others of this type. This API sit in the "implementation" package, which implies we donot want users to extend. So final may make sense.